### DISCUSSION TITLE
Merge field command take an array

### DISCUSSION THREAD

**Discussion by yeboJoseph** [craftcms/cms#16297] [Ideas] [answered]
If I upgrade Craft 4 to 5, multiple fields with the same handle will be created and a number will be appended. Running `fields/auto-merge will` merge _all_ fields with the same settings. Instead, I want to run `fields/merge`  with the second argument letting me add an array. Ex: fields/merge heading [heading1, heading2, heading4, heading5]

As of now, if I wanted to merge 20 fields into one, I'd have to run the command 20 times.

**Comment by brandonkelly** [comment]
Agree this would be nice. I’m planning on making it possible once the next Yii update is released with https://github.com/yiisoft/yii2/pull/20247.

**Reply by brandonkelly** [reply, marked as answer]
Yii 2.0.52 was released last week, so I was able to get this in for Craft 5.7 🎉 (#16719)

**Reply by brandonkelly** [reply]
Craft 5.7.0 is out with that change.
